### DISCUSSION TITLE
关于CNNEncoder的使用方式？

### DISCUSSION THREAD

**Discussion by venzhane** [PaddlePaddle/PaddleNLP#370] [Q&A] [answered]
是菜鸟，想请问一下CNNEncoder是怎么使用的，看教程只有LSTMEncoder的例子，有些看不明白

**Comment by wawltor** [comment]
您好，能不能提供一下具体教程的地址了？

**Comment by Steffy-zxf** [comment, marked as answer]
@venzhane 你好！`CNNEncoder`和`LSTMEncoder`使用方式类似。`CNNEncoder` 具体使用可以参考 https://github.com/PaddlePaddle/PaddleNLP/blob/develop/paddlenlp/models/senta.py#L447

**Comment by venzhane** [comment]
> @venzhane 你好！`CNNEncoder`和`LSTMEncoder`使用方式类似。`CNNEncoder` 具体使用可以参考 https://github.com/PaddlePaddle/PaddleNLP/blob/develop/paddlenlp/models/senta.py#L447

谢谢
